dffml/record.py

```diff
@@ -182,6 +182,28 @@ def key(self) -> str:
     def evaluated(self, results: Dict[str, Any], overwrite=False):
         """
         Updates features with the result dict
+
+        Parameters
+        ----------
+        results : Dict
```

Comment:
I think this needs to be lowercase

```suggestion
        results : dict
```